src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/xgq_hwemu.cxx

```diff
@@ -276,13 +298,33 @@ namespace hwemu {
         break;
 
       default:
-        printf("Error: Unknown command.\n");
+        std::cout << "Error: Unknown command." << std::endl;
         return -EINVAL;
     }
 
     return 0;
   }
 
+  int xgq_cmd::load_xclbin(xrt::bo& xbo, char *buf, size_t size)
+  {
+    auto data = xbo.map();
+    memcpy(data, buf, size);
+    uint32_t paddr = static_cast<uint32_t>(xbo.address());

```

Comment:
warning: use auto when initializing with a cast to avoid duplicating the type name [hicpp-use-auto]

```cpp
uint32_t paddr = static_cast<uint32_t>(xbo.address());
    ^
note: this fix will not be applied because it overlaps with another fix
```